### PR TITLE
[13.0][MIG] account_mass_reconcile

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of module account_mass_reconcile to v13.

cc ~ @ForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Code review LGTM

File: account_mass_reconcile/models/base_advanced_reconciliation.py [hunk 1]
```diff
@@ -217,9 +218,9 @@ def _skip_line(self, move_line):
         """
         return False
 
-    @api.multi
     def _rec_auto_lines_advanced(self, credit_lines, debit_lines):
         """ Advanced reconciliation main loop """
+        # pylint: disable=invalid-commit
```
Comment (MiquelRForgeFlow) [inline account_mass_reconcile/models/base_advanced_reconciliation.py:223]: why? what does it mean?

**Review by MiquelRForgeFlow — APPROVED** [review]

File: account_mass_reconcile/models/base_advanced_reconciliation.py (hunk 1 above)
Comment (AdriaGForgeFlow) [reply account_mass_reconcile/models/base_advanced_reconciliation.py:223]: To avoid Travis from failing due to commits done inside this function

**Comment by captivea-dcr** [comment]
@pedrobaeza @gurneyalex do you know what happened to this PR? (still merging since 17 days)

**Comment by pedrobaeza** [comment]
GitHub stuck notifying bot

/ocabot merge nobump

**Comment by captivea-dcr** [comment]
thanks @pedrobaeza

**Pull request merged**